.changeset/early-turkeys-hunt.md

```diff
@@ -0,0 +1,7 @@
+---
+"@paprika/confirmation": patch
```

Comment:
Isn't it a minor change?

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes